subdomains/Subdomain Alfa/src/main/scala/legacy/SparkOpInstance12.scala

```diff
@@ -0,0 +1,25 @@
+package legacy
+import org.apache.spark.sql.SparkSession
+
+import platform.common_classes.SparkOp
+import org.apache.spark.sql.DataFrame
+import platform.common_classes.Metadata
+import platform.common_classes.RunConfigurations
+
+object SparkOpInstance12 extends SparkOp {
+  val randomValue: Int = 3142 // Hardcoded random value
+
+  override def name: Input = "dataset/spark-op-instance-12"
+  override def inputs: Set[Input] = Set(SparkOpInstance11.name) // Reference to SparkOpInstance11 as an input using object name
```

Comment:
SparkOpInstance11 is not in the same Subdomain - so as I stated, it needs to be hardcoded with the name, also, Its a set of Inputs and here you are passing the name directly;